q2_longitudinal/plugin_setup.py

```diff
@@ -256,7 +256,8 @@
     inputs={'table': FeatureTable[RelativeFrequency]},
     parameters={**miscellaneous_parameters,
                 **shared_parameters,
-                'metric': Str},
+                'metric': Str,
+                'baseline': Str},
```

Comment:
Why is this a `Str`? My understanding is that this should be some kind of numerical value here, and is probably contributing to my confusion in the description below.

Reply:
Thanks for catching! I suppose I was thinking that we might want to accept non-numeric values (e.g., what if a user's "state" category consisted of the following values: `[1,2,3,4,baseline]`?)

But that thinking is hideously erroneous. If only because we already enforce that `state` is a numeric column (otherwise it might not sort correctly).

So now `baseline` is a `Float`